DNN Platform/Providers/AuthenticationProviders/DotNetNuke.Authentication.Facebook/Login.ascx.cs

```diff
@@ -69,7 +69,10 @@ protected override void AddCustomProperties(NameValueCollection properties)
         {
             base.AddCustomProperties(properties);
 
-            properties.Add("Facebook", OAuthClient.GetCurrentUser<FacebookUserData>().Link.ToString());
+            var facebookUserData = OAuthClient.GetCurrentUser<FacebookUserData>();
+            if (null != facebookUserData.Link) {
```

Comment:
I don't think we're using [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) elsewhere in the codebase, can you please switch to `facebookUserData.Link != null`?